Framework/basic.json

```diff
@@ -16,7 +16,7 @@
         "url": "infologger:///debug?qc"
       },
       "consul": {
-        "url": "http://consul-test.cern.ch:8500"
+        "url": "consul-test.cern.ch:8500"
```

Comment:
This is just a cosmetic change or it doesn't work without it? Maybe by consequence the urls in other files should be changed then?

Reply:
no reason for the change, I just changed it back and forth and forgot what was the original format. I will roll back.